sequencing-server/src/lib/codegen/ActivityTypescriptCodegen.ts

```diff
@@ -7,7 +7,7 @@ readonly startTime: Temporal.Instant;
 readonly endTime: Temporal.Instant;`;
 
 export function generateTypescriptForGraphQLActivitySchema(activitySchema: GraphQLActivitySchema): string {
-  const activityTypeAlias = `type ActivityType = ${activitySchema.name};`;
+  const activityTypeAlias = `type ActivityType = ActivityTypes.${activitySchema.name};`;
```

Comment:
Does this imply the UI needs to change the default expansion starter code? 

https://github.com/NASA-AMMOS/aerie-ui/blob/c69d54048e0469bc233efb4713cc713a74c19da8/src/components/expansion/ExpansionRuleForm.svelte#L24

Reply:
Nope! The type gets picked up already:
<img width="563" alt="Screenshot 2023-02-28 at 10 31 51 AM" src="https://user-images.githubusercontent.com/3433719/221946717-5b3340af-46df-4caa-9b38-6dc8eb27b250.png">

Reply:
I don't think so. This type alias should be ok now that we've namespaced the actual activity name that customers can modify. Now if they make a command named `ActivityType`....

Reply:
> I don't think so. This type alias should be ok now that we've namespaced the actual activity name that customers can modify. Now if they make a command named `ActivityType`....

Yeah, that would cause a big problem! I think we can cross that bridge if it ever becomes an issue 😭

Reply:
I think we should still encourage CamelCase activity types and CAPITALIZED command stems for clarity of readability as well.